[11:53:53]Error in db.py: connection to server at "localhost" (::1), port 5432 failed: FATAL:  database "lol" does not exist
[11:56:36]Error in db.py: connection to server at "localhost" (::1), port 5432 failed: FATAL:  database "lol" does not exist
[11:57:55]Error in db.py: connection to server at "localhost" (::1), port 5432 failed: FATAL:  database "lol" does not exist
[11:58:17]Error in db.py: connection to server at "localhost" (::1), port 5432 failed: FATAL:  database "lol" does not exist
[12:43:08]Error in Signal_Main.py: name 'console_process' is not defined
[15:29:07]Error in console.py: name 'console_process' is not defined
[15:30:04]Error in console.py: name 'console_process' is not defined
[15:37:08]Error in console.py: module 'tkinter' has no attribute 'scrolledtext'
[15:37:16]Error in console.py: name 'scrolledtext' is not defined
[15:50:02]Error in db.py: connection to server at "localhost" (::1), port 5432 failed: FATAL:  database "lol" does not exist
[15:51:07]Error in db.py: connection to server at "localhost" (::1), port 5432 failed: FATAL:  database "lol" does not exist
[15:51:07]Error in Signal_Main.py: division by zero
[15:51:49]Error in db.py: connection to server at "localhost" (::1), port 5432 failed: FATAL:  database "lol" does not exist
[15:51:49]Error in Signal_Main.py: division by zero
[15:53:04]Error in db.py: connection to server at "localhost" (::1), port 5432 failed: FATAL:  database "lol" does not exist
[15:53:04]Error in Signal_Main.py: division by zero
[15:53:22]Error in db.py: connection to server at "localhost" (::1), port 5432 failed: FATAL:  database "lol" does not exist
[15:53:24]Error in Signal_Main.py: [Errno 2] No such file or directory: 'c:\\Users\\Tom�\\Desktop\\Tom�\\Dokumenty\\VSCode\\Python\\PyGame\\Signal_Bunker\\files\\functions\\temp.log'
[15:55:20]Error in db.py: connection to server at "localhost" (::1), port 5432 failed: FATAL:  database "lol" does not exist
[15:55:22]Error in Signal_Main.py: [Errno 2] No such file or directory: 'temp.log'
[15:57:00]Error in db.py: connection to server at "localhost" (::1), port 5432 failed: FATAL:  database "lol" does not exist
[15:57:36]Error in db.py: connection to server at "localhost" (::1), port 5432 failed: FATAL:  database "lol" does not exist
[16:00:13]Error in db.py: connection to server at "localhost" (::1), port 5432 failed: FATAL:  database "lol" does not exist
[16:00:21]Error in Signal_Main.py: [WinError 3] The system cannot find the path specified: 'logs/temp.log'
[16:00:51]Error in db.py: connection to server at "localhost" (::1), port 5432 failed: FATAL:  database "lol" does not exist
[16:00:56]Error in Signal_Main.py: [WinError 3] The system cannot find the path specified: 'logs\\temp.log'
[16:01:33]Error in db.py: connection to server at "localhost" (::1), port 5432 failed: FATAL:  database "lol" does not exist
[16:01:35]Error in Signal_Main.py: [WinError 3] The system cannot find the path specified: '\\logs\\temp.log'
[16:04:07]Error in db.py: connection to server at "localhost" (::1), port 5432 failed: FATAL:  database "lol" does not exist
[16:04:09]Error in Signal_Main.py: [WinError 3] The system cannot find the path specified: 'c:\\Users\\Tom�\\Desktop\\Tom�\\Dokumenty\\VSCode\\Python\\PyGame\\Signal_Bunker\\files\\files\\logs\\temp.log'
[16:06:05]Error in db.py: connection to server at "localhost" (::1), port 5432 failed: FATAL:  database "lol" does not exist
[16:06:15]Error in db.py: connection to server at "localhost" (::1), port 5432 failed: FATAL:  database "lol" does not exist
[16:09:31]Error in db.py: connection to server at "localhost" (::1), port 5432 failed: FATAL:  database "lol" does not exist
[16:10:31]Error in Signal_Main.py: [WinError 2] The system cannot find the file specified: 'C:\\Users\\Tom�\\Desktop\\Tom�\\Dokumenty\\VSCode\\Python\\PyGame\\Signal_Bunker\\files\\logs\\temp.log'
[16:24:27]Error in db.py: connection to server at "localhost" (::1), port 5432 failed: FATAL:  database "lol" does not exist
[18:46:13]Error in db.py: connection to server at "localhost" (::1), port 5432 failed: FATAL:  database "lol" does not exist
[21:38:31]Error in db.py: connection to server at "localhost" (::1), port 5432 failed: FATAL:  database "lol" does not exist
[21:45:45]Error in db.py: connection to server at "localhost" (::1), port 5432 failed: FATAL:  database "lol" does not exist
[21:46:19]Error in db.py: connection to server at "localhost" (::1), port 5432 failed: FATAL:  database "lol" does not exist
[08:49:27]Error in db.py: connection to server at "localhost" (::1), port 5432 failed: FATAL:  database "lol" does not exist
[08:50:28]Error in db.py: connection to server at "localhost" (::1), port 5432 failed: FATAL:  database "lol" does not exist
[08:50:44]Error in Signal_Main.py: [WinError 2] The system cannot find the file specified: 'C:\\Users\\Tom�\\Desktop\\Tom�\\Dokumenty\\VSCode\\Python\\PyGame\\Signal_Bunker\\files\\logs\\temp.log'[09:10:17]Error in db.py: connection to server at "localhost" (::1), port 5432 failed: FATAL:  database "lol" does not exist
[09:11:04]Error in db.py: connection to server at "localhost" (::1), port 5432 failed: FATAL:  database "lol" does not exist
[09:11:04]Error in Signal_Main.py: name 'db' is not defined
[09:11:33]Error in db.py: connection to server at "localhost" (::1), port 5432 failed: FATAL:  database "lol" does not exist
[09:11:34]Error in Signal_Main.py: argument 1 must be pygame.surface.Surface, not str
[09:12:57]Error in db.py: connection to server at "localhost" (::1), port 5432 failed: FATAL:  database "lol" does not exist
[09:13:08]Error in db.py: connection to server at "localhost" (::1), port 5432 failed: FATAL:  database "lol" does not exist
[09:13:15]Error in db.py: connection to server at "localhost" (::1), port 5432 failed: FATAL:  database "lol" does not exist
[09:13:29]Error in db.py: connection to server at "localhost" (::1), port 5432 failed: FATAL:  database "lol" does not exist
[09:13:38]Error in db.py: connection to server at "localhost" (::1), port 5432 failed: FATAL:  database "lol" does not exist
[09:14:06]Error in db.py: connection to server at "localhost" (::1), port 5432 failed: FATAL:  database "lol" does not exist
[09:14:24]Error in db.py: connection to server at "localhost" (::1), port 5432 failed: FATAL:  database "lol" does not exist
[09:14:34]Error in db.py: connection to server at "localhost" (::1), port 5432 failed: FATAL:  database "lol" does not exist
[09:15:08]Error in db.py: connection to server at "localhost" (::1), port 5432 failed: FATAL:  database "lol" does not exist
[09:21:42]Error in Signal_Main.py: 'int' object is not subscriptable
[09:23:54]Error in Signal_Main.py: 'bool' object is not subscriptable
[09:47:22]Error in db.py: connection to server at "localhost" (::1), port 5432 failed: FATAL:  database "lol" does not exist
[09:47:31]Error in db.py: connection to server at "localhost" (::1), port 5432 failed: FATAL:  database "lol" does not exist
[09:48:11]Error in db.py: connection to server at "localhost" (::1), port 5432 failed: FATAL:  database "lol" does not exist
[09:48:20]Error in db.py: connection to server at "localhost" (::1), port 5432 failed: FATAL:  database "lol" does not exist
[10:17:41]Error in db.py: connection to server at "localhost" (::1), port 5432 failed: FATAL:  database "lol" does not exist
[10:20:05]Error in db.py: connection to server at "localhost" (::1), port 5432 failed: FATAL:  database "lol" does not exist
[20:38:54]Error in Signal_Main.py: name 'font' is not defined[20:40:11]Error in Signal_Main.py: name 'text_surface' is not defined[20:41:15]Error in Signal_Main.py: name 'text_surface' is not defined[20:41:39]Error in Signal_Main.py: name 'text_surface' is not defined